tests/logictest/suites/base/09_fuse_engine/09_0020_analyze

```diff
@@ -27,11 +27,9 @@ select * from t order by a;
 6
 7
 
-statement query T
+statement ok
```

Comment:
Maybe this sql need result is empty, not only ok.

Reply:
In our new logictest framework, I'll use `query ok` or `query types` without `----` and results. Both ways are ok to me.